Documentation/workflow/DevelopmentTips.md

```diff
@@ -1,37 +1,292 @@
-# Development Tips
+# Development tips and native debugging
 
 Tips and tricks while developing Xamarin.Android.
 
-# Update Directory
+## Update directory
```

Comment:
I think this and the `Attaching LLDB using ...` sections should be `#` sections, not `##` sections, as is done in `JenkinsBuildArtifacts.md`.

Reply:
Sounds good. I wasn't sure if that heading level was intentional in the original. I think the docs.microsoft.com docs use just one H1 heading for the title of the page, but for these in-repo docs, we don't need to match those guidelines.